test/honeybadger_test.exs

```diff
@@ -17,21 +18,50 @@ defmodule HoneybadgerTest do
     :meck.expect(HTTP, :post, fn(_url, _data, _headers) -> %HTTP.Response{} end)
     Application.put_env(:honeybadger, :exclude_envs, [])
 
-    url = Application.get_env(:honeybadger, :origin) <> "/v1/notices"
-    headers = [{"Accept", "application/json"},
-               {"Content-Type", "application/json"},
-               {"X-API-Key", "at3stk3y"}]
-
     defmodule Sample do
       def notify do
-        Honeybadger.notify(%RuntimeError{}, %{})
+        metadata = %{foo: "Bar"}
+        Honeybadger.notify(%RuntimeError{}, metadata)
       end
     end
 
     Sample.notify
-    :timer.sleep 250
+    :timer.sleep 50
+
+    url = Application.get_env(:honeybadger, :origin) <> "/v1/notices"
+    headers = [{"Accept", "application/json"},
+               {"Content-Type", "application/json"},
+               {"X-API-Key", "at3stk3y"}]
+
+    payload = %Notice{
+      error: %{
+        backtrace: [
+          %{context: "all", file: "lib/process.ex", method: "info", number: 384},
+          %{context: "all", file: "lib/honeybadger.ex", method: "do_notify", number: 133},
+          %{context: "all", file: "lib/task/supervised.ex", method: "do_apply", number: 74},
+          %{context: "all", file: "proc_lib.erl", method: "init_p_do_apply", number: 240}
+				], 
+        class: "RuntimeError",
+        message: "runtime error",
+        tags: []},
+      notifier: %{
+        name: "Honeybadger Elixir Notifier",
+        url: "https://github.com/honeybadger-io/honeybadger-elixir",
+        version: "0.3.0"
+      },
+      request: %{
+        context: %{
+          foo: "Bar"
+        }
+      },
+      server: %{
+        environment_name: :test, 
+        hostname: "rb-2",
+        project_root: "/Users/rb/code/honeybadger"
+      }
+    }
```

Comment:
I really wish there was an easier way to test the entire payload without needing all the specifics. This test will break if line numbers break. I need to think of a better way to test this.